osfclient/cli.py

```diff
@@ -65,6 +69,39 @@ def _setup_osf(args):
 
     return OSF(username=username, password=password)
 
+def init(args):
+    """Initialize or edit an existing .osfcli.config file
+
+    """
+    # reading existing config file, convert to configparser object
+    config = config_from_file()
+    config_ = configparser.ConfigParser()
```

Comment:
Good idea, will do that. For the `ENV` values: I assumed people don't want to prefill with ENV values as the config file is the last resort in any case if I got the structure right?